protocol/testing/testnet-dev/dev.sh

```diff
@@ -210,6 +210,10 @@ edit_config() {
 
 	# Disable pex
 	dasel put -t bool -f "$CONFIG_FOLDER"/config.toml '.p2p.pex' -v 'false'
+
+  # Enable Sidecar Prometheus
```

Comment:
nit: These metrics are being enabled on the application, they are slinky-specific app metrics, update comment accordingly